integrations/amazon_bedrock/tests/test_document_embedder.py

```diff
@@ -117,6 +121,7 @@ def test_from_dict(self, mock_boto3_session):
         assert embedder.progress_bar
         assert embedder.meta_fields_to_embed == []
         assert embedder.embedding_separator == "\n"
+        assert embedder.boto3_config == {"read_timeout": 1000}
```

Comment:
same thing here as above

Reply:
Done. I parametrized the tests for the boto3_cofig value.